src/Functions/toStartOfInterval.cpp

```diff
@@ -25,8 +27,15 @@ namespace ErrorCodes
 
 class FunctionToStartOfInterval : public IFunction
 {
+    const bool enable_default_monday_first = true;
```

Comment:
Not clear why default initialization (`= true`) is needed when the member variable is initialized in any case (l. 36).

Reply:
Let's make this more intuitive and change the variable type from `bool` to `FirstDayOfWeek`.

Reply:
Also, please give it a more intuitive name like "first_day_of_week".